### ISSUE TITLE
hierBAPS Error in names

### ISSUE THREAD

**Issue by Perugolate** [gtonkinhill/rhierbaps#1]
Any idea what is going on here?

```
R> hb.results2 <- hierBAPS(snp.matrix2, max.depth = 2, n.pops = 20, quiet = FALSE, n.cores = 12)
[1] "---- Current depth: 0 ----"
[1] "Current depth: 0 Cluster ID: 1"
 Round: 102/300 Type: 4 Log marginal likelihood: -10645.2016501294[1] "Converged locally!"
[1] "Best partition: Nclusters 4 Log(ml*prior) -10645.2016501294"
[1] "---- Current depth: 1 ----"
[1] "Current depth: 1 Cluster ID: 1"
 Round: 102/300 Type: 4 Log marginal likelihood: -757.226095444483[1] "Converged locally!"
[1] "Best partition: Nclusters 3 Log(ml*prior) -757.226095444483"
[1] "Current depth: 1 Cluster ID: 2"
 Round: 102/300 Type: 4 Log marginal likelihood: -3802.43328390813[1] "Converged locally!"
[1] "Best partition: Nclusters 2 Log(ml*prior) -3802.43328390813"
[1] "Current depth: 1 Cluster ID: 3"
[1] "Current depth: 1 Cluster ID: 4"
Error in names(lml.list[[cur.depth + 1]]) <- avail.cluster.ids :
  'names' attribute [4] must be the same length as the vector [2]
```

**Comment by gtonkinhill** [comment]
Hi, is it possible you could attach a small reproducible example? That would help me out in fixing the issue.

**Comment by Perugolate** [comment]
Thanks

```r
library(rhierbaps)
library(ggtree)
library(phytools)
set.seed(1234)

snp.matrix <- load_fasta("SNP_whole_matrix.aln")
hb.results <- hierBAPS(snp.matrix, quiet = FALSE, n.cores = 12)
head(hb.results$partition.df)
```


[SNP_whole_matrix.txt](https://github.com/gtonkinhill/rhierbaps/files/1673744/SNP_whole_matrix.txt)

**Comment by gtonkinhill** [comment]
List naming issue fixed in commit  4016fcc
